nix/wire-server.nix

```diff
@@ -442,6 +442,29 @@ let
     };
   }) cabal-plan;
 
+  sbomqs = pkgs.buildGoModule rec {
+    pname = "sbomqs";
+    version = "0.0.30";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "interlynk-io";
+      repo = "sbomqs";
+      rev = "v${version}";
+      hash = "sha256-+y7+xi+E8kjGUjhIRKNk6ogcQMP+Dp39LrL66B1XdrQ=";
+    };
+
+    vendorHash = "sha256-V6k7nF2ovyl4ELE8Cqe/xjpmPAKI0t5BNlssf41kd0Y=";
+
+    #ldflags = [ "${envLdflags}" ];
```

Comment:
Do we still need this comment?

Reply:
Nope. This is just something generated by the thing i used to generate this derivation. (It’s pretty cool, it’s called nix-init and it works fairly well for go programs)